### PR TITLE
add id() op for bytescheduler.torch.optimizer.py

### PR DESCRIPTION
**Pull request by un-knight**
fix issues #143

### CODE REVIEW

**Comment by bobzhuyb** [comment]
@pengyanghua Please review this.

**Comment by ymjiang** [comment]
Hi @un-knight, please merge into `bytescheduler` branch rather than master.

**Comment by pengyanghua** [comment]
@ymjiang The branch is correct, as the commit is for https://github.com/bytedance/byteps/tree/master/byteps/bytescheduler/torch

**Comment by bobzhuyb** [comment]
@ymjiang @pengyanghua I guess we should rename the folder to something else.... Even I was confused for a while. Maybe move it into pytorch folder.

**Comment by pengyanghua** [comment]
@bobzhuyb How about moving to byteps/byteps/torch/bytescheduler.py?

**Comment by bobzhuyb** [comment]
@ymjiang I think it will be better than where it is now. However, the name bytescheduler is still confusing... maybe rename it to nobarrier or something else. It's not really the whole bytescheduler -- it's just for crossing the global barrier. Let's just keep bytescheduler inside bytescheduler branch.

**Comment by pengyanghua** [comment]
@bobzhuyb Sure, I will create a new pull request.

**Comment by un-knight** [comment]
> @bobzhuyb Sure, I will create a new pull request.

@pengyanghua So, do I need to pull a new request which merges into the scheduler branch?

**Comment by pengyanghua** [comment]
@un-knight @bobzhuyb  After merging this request, I will create another pull request to avoid name confusion.

**Comment by bobzhuyb** [comment]
@pengyanghua Okay. I'll merge this now. Please submit a new PR that addresses what we have said, and do necessary testing.
